code/modules/power/power_store.dm

```diff
@@ -207,7 +207,7 @@
 /obj/item/stock_parts/power_store/proc/on_reagent_change(datum/reagents/holder)
 	SIGNAL_HANDLER
 
-	rigged = corrupted || !!holder.has_reagent(/datum/reagent/toxin/plasma, 5) //has_reagent returns the reagent datum
+	rigged = corrupted || holder.has_reagent(/datum/reagent/toxin/plasma, 5) //has_reagent returns the reagent datum
```

Comment:
The comment states why it's doing what it's doing, it's doing this to set rigged to true/false, what you're doing is setting it to a reagent datum.